config-model-api/src/main/java/com/yahoo/config/application/api/DeploymentSpec.java

```diff
@@ -241,7 +241,9 @@ public ZoneEndpoint zoneEndpoint(InstanceName instance, ZoneId zone, ClusterSpec
         if (   zone.environment().isTest()
             && instances().stream()
                           .anyMatch(spec -> spec.zoneEndpoints().getOrDefault(cluster, Map.of()).values().stream()
-                                                .anyMatch(endpoint -> ! endpoint.isPublicEndpoint())))
+                                                .anyMatch(endpoint -> ! endpoint.isPublicEndpoint()))
+               // Remove once Azure Private Link has been implemented
+            && !zone.region().value().startsWith("azure-"))
```

Comment:
This is done to avoid requiring additional method parameters.  Changing the method signature requires preserving the old for backwards compatibility with old models, and adding the new.